geopandas/plotting.py

```diff
@@ -747,12 +747,24 @@ def plot_dataframe(
         vmax = len(categories) - 1 if vmax is None else vmax
 
     if scheme is not None:
+        mc_err = "The 'mapclassify' >= 2.4.0 is required to use the 'scheme' keyword."
```

Comment:
```suggestion
        mc_err = "The 'mapclassify' package (>= 2.4.0) is required to use the 'scheme' keyword."
```

(or alternatively leave out the "The")